server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8 as builder
+
+COPY . /server
+WORKDIR /server
+
+RUN chmod 777 mvnw
+
+# Running from Windows will add a windows EOF. Need to change it to linux.
+RUN /bin/sed -i 's/\r//' mvnw
+
+# Build Spring project
+RUN ./mvnw install
+
+FROM openjdk:8
```

Comment:
How come you need this a second time?

Reply:
The first one is an intermediate container. That way we can pull the whole project in do all build things in the first one. And then the 2nd one is just the java environment and the jar